plugins/source/aws/policies/queries/iam/unused_creds_disabled_45_days.sql

```diff
@@ -0,0 +1,18 @@
+insert into aws_policy_results
+select
+  :'execution_time',
+  :'framework',
+  :'check_id',
+  'Ensure credentials unused for 90 days or greater are disabled (Scored)',
+  account_id,
+  arn,
+  case when
+      (password_enabled and password_last_used < (now() - '45 days'::INTERVAL)
+        or (last_used < (now() - '45 days'::INTERVAL)))
+      then 'fail'
+      else 'pass'
+  end
+from
+    aws_iam_users
+inner join aws_iam_user_access_keys
+    on aws_iam_users.cq_id = aws_iam_user_access_keys.user_cq_id
```

Comment:
This won't catch root user keys as there is no way for us to list IAM keys for the root user

Reply:
I'm not sure if there's a specific table that stores the root access keys as it's technically not an IAM User.  the AWS API call to ListAccessKeys should also list root access keys (I have to verify this).    

On another note from a naming convention, I'm going back and forth if this should be categorized as aws_iam_user_access_keys for root or should it be something like aws_root_access_keys.  (That still belongs under the broader unused_creds_disabled_45_days).  

https://docs.aws.amazon.com/IAM/latest/APIReference/API_ListAccessKeys.html

Are we storing root account information somewhere?  I briefly browsed through and saw an aws_accounts table but that did not seem quite right.  Happy to discuss this further.